script.js

```diff
@@ -0,0 +1,16 @@
+const btn = document.querySelector('#btn_hambrg');
+
+btn.addEventListener('click', () => {
+  const navbar = document.querySelector('.navbar');
+
+  navbar.style.disply = 'block';
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure the CSS property `display` is spelled correctly.
- [ ] And as you may notice this code at line 6 has no effect on the hamburger menu functionality, kindly make sure you use the `You Are not Gonna Need It(YAGNI)` method in your code base to keep it clean. You may go ahead and delete this line of code.
```suggestion
  navbar.style.display = 'block';
```